src/reactive_state_access.rs

```diff
@@ -523,3 +527,51 @@ where
         self.get_with(|s| other.get_with(|o| *o - *s))
     }
 }
+
+#[cfg(test)]
+mod test {
+    use super::*;
+    use crate::*;
+    use atomic_hooks_macros::*;
+    use store::RxFunc;
+    #[atom(undo)]
+    fn a() -> AtomUndo<i32> {
+        0
+    }
+
+    #[atom(undo)]
+    fn b() -> AtomUndo<i32> {
+        0
+    }
+
+    #[test]
+    fn test_undo() {
```

Comment:
Well, I am making tests which I will add to the doc later .

As you can see for some reason, I need 2 times to use `global_undo_queue().travel_backwards()` to make the test pass.

Is it because of get() ? If I try to println the `b().get(),` then I need even more `global_undo_queue().travel_backwards()`

Is it the expected behavior?